allennlp/predictors/bidaf.py

```diff
@@ -1,8 +1,14 @@
+# pylint: disable=protected-access
+from copy import deepcopy
+from typing import Dict, List
 from overrides import overrides
-
+import numpy as np
```

Comment:
Just `import numpy`.  Also, check the [import guidelines](https://github.com/allenai/allennlp/blob/master/STYLE.md#module-layout-and-imports).